Sources/Misc/SandboxEnvironmentDetector.swift

```diff
@@ -59,3 +61,61 @@ final class BundleSandboxEnvironmentDetector: SandboxEnvironmentDetector {
 }
 
 extension BundleSandboxEnvironmentDetector: Sendable {}
+
+private extension BundleSandboxEnvironmentDetector {
+
+    var isProductionReceipt: Bool {
```

Comment:
Maybe move this inside of the `#if os(macOS)` check too? It's potentially very inefficient for mobile devices, so that way we make sure we don't use it there.